src/renderer/components/interact/forms/Bond.tsx

```diff
@@ -41,23 +44,38 @@ export const Bond: React.FC<Props> = ({ onFinish: onFinishProp, max, isLoading =
     [onFinishProp]
   )
 
+  // graterThan returns pure function and there is no need to validate its deps
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const graterThenZero = useCallback(
+    greaterThan(ZERO_BN)(intl.formatMessage({ id: 'wallet.validations.graterThen' }, { value: 0 })),
+    [intl]
+  )
+
+  // lessThanOrEqualTo returns pure function and there is no need to validate its deps
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const lessOrEqualToMax = useCallback(
+    lessThanOrEqualTo(max.amount())(
+      intl.formatMessage(
+        { id: 'wallet.validations.lessThen' },
+        { value: formatAssetAmount({ amount: max, decimal: 8, trimZeros: true }) }
+      )
+    ),
+    [max, intl]
+  )
```

Comment:
No need to put these into another Callback - all can be directrly go it into `amountValidator` (similar to [validateTxAmountInput](https://github.com/thorchain/asgardex-electron/blob/1ca406c9fb60eab1a41008623faa81b8330f536d/src/renderer/components/wallet/txs/TxForm.util.ts#L18))